### PR TITLE
ShwetaD_Assignment_13and14

### PR DESCRIPTION
**Pull request by ShwetaDharmadhikari**
image

image

image

### CODE REVIEW

**Review by harshvegada — APPROVED** [review]
Code Logic looks good to me!

File: src/shweta_Dharmadhikari/array_examples/CountNumbersFromArray.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package shweta_Dharmadhikari.array_examples;
+
+public class CountNumbersFromArray {
+	int count = 0;
+
+	int countOfPositiveNumber() {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/array_examples/CountNumbersFromArray.java:6]: take input array as parameter in the method

File: src/shweta_Dharmadhikari/array_examples/CountNumbersFromArray.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		int[] input = { 1, -11, -44, 23, 55, -99, -23, -22 };
+		for (int index = 0; index < input.length; index++) {
+			if (input[index] >= 0) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	int countNumberOfZeros() {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/array_examples/CountNumbersFromArray.java:16]: take input array as parameter in the method

File: src/shweta_Dharmadhikari/array_examples/DifferenceBetweenMinAndMaxNumber.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package shweta_Dharmadhikari.array_examples;
+
+public class DifferenceBetweenMinAndMaxNumber {
+	int[] input = { 1, 11, 44, 23, 55, 99, 23, 22 };
+	int index = 0;
+	int max = input[index];
+	int min = input[index];
+
+	int diffBetweenMaxOddAndMaxEvenNumber() {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/array_examples/DifferenceBetweenMinAndMaxNumber.java:9]: take input array as parameter in the method

File: src/shweta_Dharmadhikari/array_examples/DifferenceBetweenMinAndMaxNumber.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+		for (; index < input.length; index++) {
+			if (input[index] > max && input[index] % 2 == 0)
+				max = input[index];
+			else if (input[index] > min && input[index] % 2 != 0)
+				min = input[index];
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/array_examples/DifferenceBetweenMinAndMaxNumber.java:14]: Please use a suitable variable name. Here min variable name will confuse the reviewer.

File: src/shweta_Dharmadhikari/array_examples/MaxMinCountFromArray.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
+package shweta_Dharmadhikari.array_examples;
+
+public class MaxMinCountFromArray {
+
+	int[] input = { 1, 11, 44, 23, 55, 99, 23 };
+
+	int maxNumber() {
```
Comment (Upasana1703) [inline src/shweta_Dharmadhikari/array_examples/MaxMinCountFromArray.java:7]: take input array as parameter in the method

**Review by Upasana1703 — APPROVED** [review]
Code looks good to me

**Pull request merged**